ompi/mca/op/avx/configure.m4

```diff
@@ -315,15 +315,15 @@ AC_DEFUN([MCA_ompi_op_avx_CONFIG],[
                        [$op_sse3_support],
                        [SSE3 supported in the current build])
     AM_CONDITIONAL([MCA_BUILD_ompi_op_has_avx512_support],
-                   [test "$op_avx512_support" == "1"])
+                   [test "$op_avx512_support" = "1"])
```

Comment:
This patch doesn't seem to have anything to do with other commits in this PR.

Reply:
Sure, I'll spit it out. I didn't do it prematurely given it's trivial, but it's no problem.

Reply:
Split into https://github.com/open-mpi/ompi/pull/11083.